client/src/commands.ts

```diff
@@ -518,3 +682,41 @@ function isDenoDisabledCompletely(): boolean {
     vscode.workspace.getConfiguration(EXTENSION_NS, f)
   ).every(isScopeDisabled);
 }
+
+const UNSUPPORTED_COMMON_COMPILER_OPTIONS = [
+  "baseUrl",
+  "composite",
+  "declaration",
+  "declarationDir",
+  "declarationMap",
+  "emitBOM",
+  "emitDeclarationOnly",
+  "emitDecoratorMetadata",
+  "generateCpuProfile",
+  "importHelpers",
+  "incremental",
+  "inlineSourceMap",
+  "inlineSources",
+  "moduleResolution",
+  "newLine",
+  "noEmit",
+  "noEmitHelpers",
+  "noEmitOnError",
+  "outDir",
+  "outFile",
+  "paths",
+  "preserveSymlinks",
+  "preserveWatchOutput",
+  "removeComments",
+  "rootDir",
+  "rootDirs",
+  "skipLibCheck",
+  "sourceMap",
+  "sourceRoot",
+  "stripInternal",
+  "tsBuildInfoFile",
+  "typeRoots",
+  "watch",
+  "watchDirectory",
+  "watchFile",
```

Comment:
I don't really like the idea of having another list of compiler options here, but I can't think of anything else and this is only used for the conversion.